Assets/SEE/Controls/Actions/ResizeNodeAction.cs

```diff
@@ -468,184 +578,211 @@ void StartResizing()
             /// </summary>
             void UpdateSize()
             {
-                Raycasting.RaycastLowestNode(out RaycastHit? targetObjectHit, out Node hitNode, nodeRef);
-                if (!targetObjectHit.HasValue)
+                Raycasting.RaycastLowestNode(out RaycastHit? targetObjectHit, out _, nodeRef);
+                if (!targetObjectHit.HasValue && !currentResizeStep.Up)
                 {
                     return;
                 }
 
-                // Collect siblings
-                Transform parent = transform.parent;
-                // We use an initial size of the list so that the memory does not need to get
-                // reallocated each time an item is added.
-                List<Transform> siblings = new(parent.childCount);
-                foreach (Transform sibling in parent)
+                // Calculate new scale and position
+                Vector3 cursorMovement;
+                if (currentResizeStep.Up)
                 {
-                    if (sibling != transform && sibling.gameObject.IsNodeAndActiveSelf())
-                    {
-                        siblings.Add(sibling);
-                    }
+                    cursorMovement = new(0f, (currentResizeStep.InitialMousePosition.y - Input.mousePosition.y) / Screen.height, 0f);
```

Comment:
Can this method also be called on VR players? If so, we cannot use the mouse position/screen height.

Reply:
I've added an issue #806 which tracks this problem, so we can merge this before handling that part.